tests/integration/test_repository.py

```
@@ -0,0 +1,159 @@
+        coupon_rate=0.028,
+        maturity_date="2030-01-01",
+        credit_rating="AA",
+        issuer_id="PARENT",
+    )
+
+    repo.upsert_asset(parent_equity)
+    repo.upsert_asset(bond)
+    session.commit()
+    session.expire_all()
+
+    repo.add_or_update_relationship("PARENT", "PARENT_BOND", "test", 0.5, False)
```

Ruff (0.14.3)

Boolean positional value in function call

(FBT003)